urts/sys/types/build.rs

```diff
@@ -1,37 +1,28 @@
 // Copyright (c) 2022 The MobileCoin Foundation
 
-//! Builds the FFI bindings for the untrusted side of the Intel SGX SDK
-use bindgen::{callbacks::ParseCallbacks, Builder};
+//! Builds the FFI type bindings for the untrusted side of the Intel SGX SDK
 use std::{env, path::PathBuf};
 
-#[derive(Debug)]
-struct Callbacks;
-
-impl ParseCallbacks for Callbacks {
-    fn item_name(&self, name: &str) -> Option<String> {
-        match name {
-            "_attributes_t" => Some("sgx_attributes_t".to_owned()),
-            "_sgx_misc_attribute_t" => Some("sgx_misc_attribute_t".to_owned()),
-            _ => None,
-        }
-    }
-}
+const URTS_TYPES: &[&str] = &[
```

Comment:
What about the `_sgx_kss_config_t` type which is actually defined in the `sgx_urts.h` header? (And transitively, `sgx_config_svn_t`.) Shouldn't that be provided here?

Reply:
good catch on `_sgx_kss_config_t` should be there now in 9c207eb.
The `sgx_config_svn_t` is coming from the core types, so brought it into the `lib.rs` for the `sgx_kss_config_t`